src/main/java/com/icthh/xm/ms/dashboard/util/FileUtils.java

```diff
@@ -0,0 +1,29 @@
+package com.icthh.xm.ms.dashboard.util;
+
+import lombok.SneakyThrows;
+import lombok.experimental.UtilityClass;
+import org.springframework.core.io.ClassPathResource;
+
+import java.io.InputStream;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+
+@UtilityClass
+public class FileUtils {
```

Comment:
Class is used for tests only. 
suggest to move to test Folder.

Reply:
It's not until for tests. Tomorrow it may be needed somewhere else. Do not see any reason for moving it to the test.

Reply:
please move utility class to test package. If tomorrow we will need to use it in the main sources we can do this.

Reply:
)))